sys/include/crypto/ciphers.h

```diff
@@ -126,6 +126,9 @@ typedef struct {
  * @param cipher_id  cipher algorithm id
  * @param key        encryption key to use
  * @param key_size   length of the encryption key
+ *
+ * @return  1 if the initialization was successful or 0 in case of an error. 
```

Comment:
`cipher_init` returns `CIPHER_ERR_INVALID_KEY_SIZE` on error, which is defined as `-3`. Just using `CIPHER_ERR_INVALID_KEY_SIZE` properly in the doc should generate a link to the definition. It would probably make sense to get rid of the "hardcoded" `0` return values and use `CIPHER_ERR_INVALID_KEY_SIZE` everywhere.